scheduler/config-k8s.edn

```diff
@@ -49,6 +59,9 @@
            :user-metrics-interval-seconds 60}
  :nrepl {:enabled? true
          :port #config/env-int "COOK_NREPL_PORT"}
+ :pools {:default "k8s-alpha"
```

Comment:
Default is historically gamma, but I'm fine if we only test with pools from now on.

Reply:
will update